File: Actividad_4/python/act_4.py
# Función que toma dos números y devuelve un resultado especial
def custom_operation(a, b):
    # Calcula el producto y la suma
    product = a * b  # Multiplicación
    sum_result = a + b  # Suma
    # Retorna el resultado concatenado
    return str(product) + str(sum_result)

# Pruebas de la función
print(custom_operation(3, 2))  # "63"
print(custom_operation(4, 8))  # "3212"

File: Actividad_4/js/act_4.js
// Función que toma dos números y devuelve un resultado especial
function customOperation(a, b) {
    // Calcula el producto y la suma
    var product = a * b; // Multiplicación
    var sum = a + b;     // Suma
    // Retorna el resultado concatenado como texto
    return product.toString() + sum.toString();
}

// Pruebas de la función
console.log(customOperation(3, 2)); // "63"
console.log(customOperation(4, 8)); // "3212"

